preview/MsixCore/msixmgr/msixmgr.cpp

```diff
@@ -105,21 +105,33 @@ int main(int argc, char * argv[])
 
             shared_ptr<IInstalledPackage> packageInfo;
             HRESULT hr = packageManager->FindPackage(cli.GetPackageFullName(), packageInfo);
-            if (packageInfo == NULL || FAILED(hr))
+
+            if (packageInfo == NULL || hr == ERROR_NOT_FOUND)
             {
-                std::wcout << std::endl;
-                std::wcout << L"No packages found " << hr << std::endl;
-                std::wcout << std::endl;
+                hr = packageManager->FindPackageByFamilyName(cli.GetPackageFullName(), packageInfo);
+                if (FAILED(hr))
+                {
+                    std::wcout << L"Failed to determine findpackage results" << hr << std::endl;
+                }
+                else if (packageInfo == NULL || hr == ERROR_NOT_FOUND)

```

Comment:
>ERROR_NOT_FOUND [](start = 54, length = 15)

I'd expect ERROR_NOT_FOUND to imply packageInfo == NULL. do you need to separately check this? If you do, this should be HRESULT_FROM_WIN32(ERROR_NOT_FOUND), as FindPackage is supposed to return HRESULT type. 
Also, if you need to check against ERROR_NOT_FOUND explicitly, this should be done before FAILED(hr)

Reply:
I do not need to check explicitly for ERROR_NOT_FOUND. packageInfo == NULL should imply the same. I just included that to cover the additional case since the method returns that error. Will remove the check.

Reply:
Removed the ERROR_NOT_FOUND check.

Reply:
Modified the if else conditions to check for if (hr == HRESULT_FROM_WIN32(ERROR_NOT_FOUND)) first. Since packageInfo would always be NULL even if FAILED(hr), removed the packageInfo==NULL check and replaced it with the ERROR_NOT_FOUND check so that it goes into the else condition too.